bootable-jar/runtime/src/main/java/org/wildfly/core/jar/runtime/BootableJar.java

```diff
@@ -230,6 +230,17 @@ private LogContext configureLogContext() throws IOException {
     }
 
     public void run() throws Exception {
+        Path script = arguments.getCLIScript();
+        if (script != null) {
+            Path markerDir = environment.getJBossHome().resolve("cli-boot-hook-dir");
+            Path outputFile = environment.getJBossHome().resolve("cli-boot-hook-output-file.txt");
+            Files.createDirectories(markerDir);
```

Comment:
being consistent with S2I is more important but we should avoid creating these types of dirs directly in `$JBOSS_HOME`. the `data` directory is a better location for such resources.

Reply:
@jmesnil , you mean <home>/standalone/data? Wondering if <home>/standalone/tmp would be more appropriate?